src/parse.rs

```diff
@@ -114,8 +114,8 @@ pub enum Tag<'a> {
     Strong,
     Code,
 
-    /// A link. The first field is the destination URL, the second is a title
-    Link(Cow<'a, str>, Cow<'a, str>),
+    /// A link. The first field is the destination URL, the second is a title, third is autolink indicator
```

Comment:
I'm not convinced the rendering is different for autolinks; I strongly suspect that the differences are in parsing. I haven't dug into this too deeply though.